ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -41,3 +41,7 @@
 
 - name: Package build images inside container image
   ansible.builtin.import_tasks: package.yml
+
+- name: Push container images to quay.rdoproject.org
+  when: push_containers | default(false) | bool
```

Comment:
I think we should prefix this var with `cifmw_edpm_build_images_` just like rest of the vars used in this role.

Also, default value can be passed in ci_framework/roles/edpm_build_images/defaults/main.yml

Reply:
renamed this var